rabbitmq/src/urabbitmq/consumer_base_impl.cpp

```diff
@@ -98,15 +98,20 @@ void ConsumerBaseImpl::OnMessage(const AMQP::Message& message,
   std::string trace_id = message.headers().get("u-trace-id");
   std::string message_data{message.body(), message.bodySize()};
 
+  ConsumedMessage consumed{
+      .message = std::string(message.body(), message.bodySize()),
```

Comment:
This is C++20 feature (designated initializers that is), we aim to support C++17